src/routes/ExercisesScreens.tsx

```diff
@@ -38,6 +47,19 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenProps): JSX.Eleme
       })
     }
   }
+  
+useFocusEffect(
+  React.useCallback(()=>{
+    AAsyncStorage.getItem('level').then((value)=>{
```

Comment:
move this function to `AsyncStorage.ts` use async fkt with Promise

Reply:
No need for you @abdullahOde-117 to write AsyncStorage-Function just use a dummy variable and add a comment, that it will be done in LUN-131. See how bahaa did it here (line 38): https://github.com/digitalfabrik/lunes-app/pull/267/files/#diff-e32395b665dac2ea67c45f7e2de4b1f284a10026c8eaa71b5da1ea046567614f